Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:12
+
+# update and upgrade libraries
+RUN apt-get -qq update \
+    && apt-get -qq upgrade \
+	&& rm -rf /var/lib/apt/lists/ \
+	&& apt-get -qq clean
+
+# create mitodl user
+RUN adduser --disabled-password --gecos "" mitodl
+ENV MITODL_USER="$(id mitodl -u):$(id mitodl -g)"
+
+# create working directory
+RUN mkdir /usr/share/app
+WORKDIR /usr/share/app
+RUN chown -R mitodl:mitodl /usr/share/app
```

Comment:
If we want to share the whole directory we should just make it an empty folder and leave the rest as is. Can we remove the lines from here down? I'll suggest related changes for `docker-compose.yml` below.